### ISSUE TITLE
Really High CPU Usage

### ISSUE THREAD

**Issue by alanpaschoal** [rustdesk/doc.rustdesk.com#300]
Hi Guys!

I might have done something wrong, I dont know...  But I have rust in 2 desktop, both with problem. I get a super high usage, normally about 50% cpu been Desktop Window Manager the responsable. But this is just when connected to  PC using rust.

![image](https://github.com/rustdesk/doc.rustdesk.com/assets/145598451/82499e6b-b217-4e5b-8893-c0d268fde909)

**Comment by spiralshapeturtle** [comment]
<img width="437" alt="image" src="https://github.com/rustdesk/doc.rustdesk.com/assets/16855935/45ce754a-17a0-463c-9942-e54d7ccbb2c8">

I have got the same issue. Are you running a virtualized host via proxmox?

If I look into proxmox VM stats I can see exactly the connection timestamps from the client, increasing the CPU by 20% thats seems to be a lot for only showing the screen.

<img width="1081" alt="image" src="https://github.com/rustdesk/doc.rustdesk.com/assets/16855935/a18d4200-5b8c-472b-9b43-89ad5a2b8d76">

**Comment by alanpaschoal** [comment]
Not really, just using on windows 10 and 11. Had to unistall from all machines.

**Comment by Rwin1234** [comment]
Not sure if this is solved in the mean time, but I got here via google with the phrase "rustdesk server uses 20% cpu time". My solution was to untick "Privacy mode" on the client. It is in the monitor settings menu. When you turn privacy mode on, the rustdesk server side immediately takes 20% CPU extra continuously.

Dont know if its relevant, but my server is a headless windows 11 PC with a fairly recent 6-core I5 CPU.

**Comment by maravento** [comment]
It's crazy. Rustdesk 1.4.1 isn't usable. 
On an HPE proliant ml110 g9 Intel Xeon CPU E5-2603 v4 1.70GHz, 64 GB DDR4 RAM, ssd 500 GB, almost 100% of the CPU reaches, when I connect remotely through this application.

<img width="1262" height="59" alt="Image" src="https://github.com/user-attachments/assets/d3716701-dc18-4764-94d3-a2a26470dda8" />
